packages/tabs/src/Tabs.tsx

```diff
@@ -234,6 +234,7 @@ function Tabs({
     const { disabled, onClick } = child.props;
 
     const tabProps = {
+      ...child.props,
       disabled,
```

Comment:
we can remove `disabled` since it's already in `child.props`

Reply:
👍